### ISSUE TITLE
Interactions system breaks when initializing SteamVR manually

### ISSUE THREAD

**Issue by eiko** [ValveSoftware/steamvr_unity_plugin#403]
I'm using the SteamVR.Extras manual initialization mentioned here: #282 and experiencing an issue where I cannot pick up a throwable object. The hover highlight still works, but grabbing it does nothing. 

I believe the issue relates specifically to activating the Player rig. One strange thing I noticed is that after the rig is activated, the SteamVR_Input message letting me know when my headset is put on / removed stops firing. In my code:

```csharp
XRSettings.LoadDeviceByName("OpenVR");

yield return new WaitUntil(() => (XRSettings.loadedDeviceName == "OpenVR"));

XRSettings.enabled = true;
Valve.VR.SteamVR.Initialize(true);
Valve.VR.InteractionSystem.Player player = Valve.VR.InteractionSystem.Player.instance;
// message appears when i put on / take off my headset
player.ActivateRig(player.rigSteamVR);
// message stops firing
```
I'm guessing there's some functionality in the SteamVR_Input system that needs to be manually initialized, but I haven't found it yet. I'll keep you updated if I find more information.

**Comment by eiko** [comment]
I got to the root of the issue which is that there were 2 instances of the `[SteamVR]` GameObject. `SteamVR.Initialize()` looks for existing objects before creating a new one, but since the player rig is disabled at first, its SteamVR instance couldn't be found.

Ultimately I discovered that SteamVR doesn't need to be initialized. Just activating the rig seems to work perfectly: 

```csharp
 XRSettings.LoadDeviceByName("OpenVR");
yield return new WaitUntil(() => (XRSettings.loadedDeviceName == "OpenVR"));

XRSettings.enabled = true;
Valve.VR.InteractionSystem.Player player = Valve.VR.InteractionSystem.Player.instance;
player.ActivateRig(player.rigSteamVR);
```

I'm not sure if this is meant to work, or if it's a coincidence and it leaves SteamVR in a fragile state.
